hsingyuc/portfolio
# Add hangtime project

**Screenshot**
<img width="1430" alt="螢幕快照 2020-05-12 下午2 27 37" src="https://user-images.githubusercontent.com/56378160/81681489-fa921080-945c-11ea-8141-1c51d2819cde.png">

<img width="160" alt="螢幕快照 2020-05-12 下午2 27 55" src="https://user-images.githubusercontent.com/56378160/81681536-01208800-945d-11ea-8ac9-2582d3db98da.png">


**Testing**
1. Check if projects section render 3 projects.
1. Click 'view code' see if connect to Github.
